app/pages/system/networking/IpPoolPage.tsx

```diff
@@ -101,32 +119,35 @@ function UtilizationBars() {
   if (ipv4.capacity === 0 && ipv6.capacity === 0n) return null
 
   return (
-    <div className="-mt-8 mb-8 flex min-w-min flex-col gap-3 lg+:flex-row">
-      {ipv4.capacity > 0 && (
-        <CapacityBar
-          icon={<IpGlobal16Icon />}
-          title="IPv4"
-          provisioned={ipv4.allocated}
-          capacity={ipv4.capacity}
-          provisionedLabel="Allocated"
-          capacityLabel="Capacity"
-          unit="IPs"
-          includeUnit={false}
-        />
-      )}
-      {ipv6.capacity > 0 && (
-        <CapacityBar
-          icon={<IpGlobal16Icon />}
-          title="IPv6"
-          provisioned={ipv6.allocated}
-          capacity={ipv6.capacity}
-          provisionedLabel="Allocated"
-          capacityLabel="Capacity"
-          unit="IPs"
-          includeUnit={false}
-        />
-      )}
-    </div>
+    <>
+      <h2 className="mb-3 pl-3 text-mono-xs text-secondary">Utilization</h2>
```

Comment:
This heading is teensy tiny. I think it should be `text-mono-sm`. We don't use `xs` in many spots.